src/routes/Devilstable.svelte

```diff
@@ -71,45 +64,101 @@
 		}
 		return fractionData;
 	});
+	/** @type {{label: string, values: boolean[]}[]} */
 	let boolData = $derived([
-		...noFractions.map((d) => {
-			/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+		...inputChipValueLabels.map((d) => {
+			const info = data.find((c) => c.label === d);
+			if (info) {
```

Comment:
for better readability consider checking for `d === 'fr'` (and `d === 'Fassung') first and then introduce the info inside the else (or default if switch-statement). Like so, `info` is not (mis?)used for two purposes (1) a switch (relying on it's nullishness if undefined) and (2) as an object array.